src/jackdaw/client.clj

```diff
@@ -305,3 +301,8 @@
                         (map (fn [^PartitionInfo x]
                                (TopicPartition. (.topic x) (.partition x)))))]
     (apply assign consumer partitions)))
+
+(defn assignment
+  "Return the assigned topics and partitions of a consumer."
+  [^KafkaConsumer consumer]
+  (map jd/datafy (.assignment consumer)))
```

Comment:
Why did you move this to the bottom?

Reply:
dont exactly recall doing that :P was not intentional

Reply:
To keep it together with the function that lets you set the assignments.